src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/WebResource.java

```diff
@@ -58,9 +61,10 @@
 
     @ApiModelProperty(position = 3, required = true)
     @NotNull
-    @URL
```

Comment:
Could you please also removed the `import` for this?

Reply:
done